src/components/profile/librariesTable/LibrariesTable.js

```diff
@@ -56,16 +66,30 @@ function TableItem({ agency, libraryName, type }) {
 export default function LibrariesTable({ data }) {
   return (
     <>
-      <Row className={styles.tableContainer}>
+      <div className={styles.headerRow}>
+        <Text className={styles.headerItem}>
+          {Translate({ context: "profile", label: "libraries" })}
+        </Text>
+        <Text className={styles.headerItem}>
+          {Translate({ context: "profile", label: "libraryType" })}
+        </Text>
+      </div>
+      <div className={styles.tableContainer}>
         {data?.map((item) => (
-          <TableItem
-            key={item.agency}
-            agency={item.agency}
-            libraryName={item.libraryName}
-            type={item.type}
-          />
+          <TableItem key={item.agencyName} {...item} />
         ))}
-      </Row>
+      </div>
     </>
   );
 }
+
+/**
+ *
+ * @param {*} agencyID
+ * @returns returns true if public library (Folkebibliotek)
+ */
+const isPublicLibrary = (agencyID) => {
+  const faroIslandsLibraries = ["900455", "911116", "911130"];
+  const parsedID = agencyID + "";
+  return parsedID?.charAt(0) === "7" || faroIslandsLibraries.includes(parsedID);
+};
```

Comment:
Forstår ikke hvorfor denne function hoister, men hvis det virker så fair nok.

Jeg synes stadigvæk, at vi skal have et felt i FBI-api som fortæller hvilken form for bibliotek vi har med at gøre i stedet for det her